src/sections/Community/Newcomers-guide/index.js

```diff
@@ -76,7 +78,7 @@ const NewcomersGuide = () => {
         </div>
         <div className="instructions">
           <Row className="step">
-            <Col sm={12} lg={6}>
+            <Col sm={12} lg={6} >
               <img src={CommunityImage1} className="finding-an-issue" alt="Community Image" />
```

Comment:
You missed this one 😅

Reply:
I thought we are not considering SVGs 😅 , will do that, thanks for pointing it out !!